wordpress-mobile/WordPress-iOS
# [News card] Notification in tab bar

Implements #9921 

This PR also should warp up the feature.

![simulator screen shot - iphone 8 - 2018-09-03 at 14 25 28](https://user-images.githubusercontent.com/2722505/44970803-d7c01100-af85-11e8-8c89-38c30479c253.png)
![simulator screen shot - iphone 8 - 2018-09-03 at 14 22 28](https://user-images.githubusercontent.com/2722505/44970811-ddb5f200-af85-11e8-9cfa-6810ac946af5.png)

In this PR: 
- The Reader icon in the tab bar should show a badge when the reader card is available (and not yet seen)
- The badge should disappear as soon as the card is presented.
- The badge should not be visible once the card is dismissed, even if the app is restarted.

As part of this PR, there are also some tweaks to the layout of the card. 

To test:
- Launch the app. Notice the badge in the reader tab
- Navigate to the reader tab. The badge should disappear
- Dismiss the card.
